src/GameSync.Models/Users/ValueObjects/Password.cs

```diff
@@ -0,0 +1,18 @@
+﻿namespace GameSync.Models.Users.ValueObjects;
+public class Password
+{
+    public string Value { get; private set; }
+
+    private Password(string value)
+    {
+        if (string.IsNullOrEmpty(value))
+            throw new ArgumentNullException(nameof(Password));
+
+        if (value.Length < 8)
+            throw new ArgumentException("Password is too short.", nameof(Password));
```

Comment:
Where are you catching this exception? Are you sure this exception won't stop application?

Reply:
Nevermind, I see it's handled